src/tv2-common/helpers/adLibNames.ts

```diff
@@ -0,0 +1,11 @@
+export function localSourceFullAudioName(source: string) {
+	return `EVS ${source} 100%`
+}
+
+export function localSourceVoAudioName(source: string) {
+	return `EVS ${source} VO`
+}
+
+export function localSourceName(source: string, vo: boolean) {
```

Comment:
Since the name evaluates to 'EVS ...' then maybe we should call the method something that indicates that? For example 'EvsAudioName'?

Same for the other two functions